app/assets/javascripts/templates/story_link.ejs

```diff
@@ -0,0 +1,11 @@
+<%= content %>
+
+<% if (icon) { %>
```

Comment:
What do you think about using React for that? Ping @eddiecode about how to do that, we did that on a prev task.